Makefile

```diff
@@ -1,3 +1,4 @@
+PY?=python
```

Comment:
inherited from the OSL and CASS pelican makefiles, but not actually used, I'll take it out